definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/homer:0.1"
```

Comment:
Not a requirement, but it might be nice to tag this with the homer release version instead (currently 4.10, I think)

Reply:
Changed it